lib/modules/manager/helmfile/utils.ts

```diff
@@ -0,0 +1,50 @@
+import is from '@sindresorhus/is';
+import { loadAll } from 'js-yaml';
+import { logger } from '../../../logger';
+import { regEx } from '../../../util/regex';
+import type { Doc } from './types';
+
+function extractYaml(content: string): string {
+  // regex remove go templated ({{ . }}) values
+  return content
+    .replace(regEx(/{{`.+?`}}/gs), '')
+    .replace(regEx(/{{.+?}}/g), '');
+}
+
+export function loadDocs(content: string): Doc[] {
+  return loadAll(extractYaml(content), null, { json: true }) as Doc[];
+}
+
+/** Looks for kustomize specific keys in a helmfile and returns true if found */
+export function areKustomizationsUsed(
```

Comment:
i think we should do this at extraction time and store inside `managerData` field, so we don't need to parse files twice.

Reply:
Of course, that is much cleaner, updated the PR now! PS: @viceice Thanks again for the timely and helpful reviews <3